src/index.html

```diff
@@ -7,9 +7,18 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
 	<link rel="icon" type="image/png" sizes="32x32" href="assets/images/favicons/favicon.png">
-	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
-	<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.1.3/Chart.bundle.min.js"></script>
+
+	<!-- Meta Tags for SEO -->
+	<meta name="keywords" content="loklak search open data source knowledge social media">
```

Comment:
There is no need to do so keywords are space separated. Also they are not important. I will remove them as they are not used by any search engine.